eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -61,6 +91,7 @@ if ($resp.Count -gt 0) {
 
     # setting variable to reference the pull request by number
     Write-Host "##vso[task.setvariable variable=Submitted.PullRequest.Number]$($resp[0].number)"
+    AddLabels $resp[0].number
```

Comment:
As long as nothing bad happens if you try to add the same label more then once I'm fine with doing this if the PR already exists.

Reply:
I have tested the scenarios. Nothing bad happen. 
You can test it by execute the pipeline here:
https://dev.azure.com/azure-sdk/playground/_build/results?buildId=500158&view=results

The PR created is here: https://github.com/Azure/azure-sdk-tools/pull/883